ggml/src/CMakeLists.txt

```diff
@@ -1167,10 +1168,7 @@ if (MATH_LIBRARY)
     target_link_libraries(ggml PRIVATE ${MATH_LIBRARY})
 endif()
 
-add_library(ggml_static STATIC $<TARGET_OBJECTS:ggml>)
-
 if (BUILD_SHARED_LIBS)
     set_target_properties(ggml PROPERTIES POSITION_INDEPENDENT_CODE ON)
-    add_library(ggml_shared SHARED $<TARGET_OBJECTS:ggml>)
-    target_link_libraries(ggml_shared PRIVATE Threads::Threads ${GGML_EXTRA_LIBS})
+    target_link_libraries(ggml PRIVATE Threads::Threads ${GGML_EXTRA_LIBS})
```

Comment:
This is probably not necessary here, it is already done in line 1164.

Reply:
Force-pushed a fix and I also think the Kompute build should work now